airflow/contrib/hooks/opsgenie_alert_hook.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import json
+
+import requests
+
+from airflow.hooks.http_hook import HttpHook
+from airflow import AirflowException
+
+
+class OpsgenieAlertHook(HttpHook):
+    """
+    This hook allows you to post alerts to Opsgenie.
+    Accepts a connection that has an Opsgenie API key as the connection's password.
+    This hook sets the domain to conn_id.host, and if not set will default
+    to ``https://api.opsgenie.com``.
+
+    Each Opsgenie API key can be pre-configured to a team integration.
+    You can override these defaults in this hook.
+
+    :param opsgenie_conn_id: The name of the Opsgenie connection to use
+    :type opsgenie_conn_id: str
+    :param payload: Opsgenie API Create Alert payload values
+        See https://docs.opsgenie.com/docs/alert-api#section-create-alert
+    :type payload: dict
+
+    """
+    def __init__(self,
+                 opsgenie_conn_id='opsgenie_default',
+                 payload={},
+                 proxy=None,
```

Comment:
```suggestion
                 proxy=None,
```

Could you also remove `payload` from being an attribute on the object and just pass it in to the execute function please?

Reply:
Fixed, and also removed `proxy` which I left behind by mistake.